src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -123,10 +123,13 @@ public bool IgnoreNullValues
         }
 
         /// <summary>
-        /// Determines whether read-only properties are ignored during serialization and deserialization.
+        /// Determines whether read-only properties are ignored during serialization.
         /// A property is read-only if it contains a public getter but not a public setter.
         /// The default value is false.
         /// </summary>
+        /// <remarks>
+        /// Read-only properties will not get deserialized regardless of this setting.
```

Comment:
Nit: "will not get" => "will not be"

Reply:
@krwq thanks for making this change!

I have two comments:

1. According to the [voice and tone rules for MS Documentation](https://github.com/dotnet/docs/blob/master/styleguide/voice-tone.md#avoid-future-tense), we should avoid future tense. So my suggestion would be to rewrite this text to:

```suggestion
        /// Read-only properties are not deserialized regardless of this setting.
```

2. I have a [PR out for review](https://github.com/dotnet/dotnet-api-docs/pull/2471/files?file-filters%5B%5D=) in dotnet-api-docs that covers most of the newest System.Text.Json triple slash comments. Strangely, the PR does not contain anything from System.Text.Json.Serialization, and the latest version of JsonSerializerOptions.xml indicates [this remark is currently empty](JsonSerializerOptions.xml). So let me know when you check-in this PR, and I'll be happy to run my tool to automatically port the comments from Json.Serialization to Docs.